tests/integration/ha_tests/helpers.py

```diff
@@ -498,6 +518,22 @@ def restore_network_for_unit(machine_name: str) -> None:
     subprocess.check_call(restore_network_command.split())
 
 
+def restore_network_for_unit_without_ip_change(machine_name: str) -> None:
+    """Restore network from a lxc container (without causing the change of the unit IP address).
+
+    Args:
+        machine_name: lxc container hostname
+    """
+    limit_set_command = f"lxc config device set {machine_name} eth0 limits.egress="
+    subprocess.check_call(limit_set_command.split())
+    limit_set_command = f"lxc config device set {machine_name} eth0 limits.ingress="
+    subprocess.check_call(limit_set_command.split())
+    limit_set_command = f"lxc config set {machine_name} limits.network.priority="
+    subprocess.check_call(limit_set_command.split())
+    override_command = f"lxc config device remove {machine_name} eth0"
```

Comment:
hm why we remove interface?

Reply:
Hmm, it's not needed. It's something that I used to test only. I'll remove it.

Reply:
Removed on https://github.com/canonical/postgresql-operator/pull/124/commits/4233128a2b838a3f42326e102af5ad984fc8d8ed.